<launch>
    <!-- circle detector node -->
    <node name="circle_detector" pkg="circle_detector" type="circle_detector.py" output="screen"/>

    <!-- camera laptop node -->
    <node name="laptop_camera_ros" pkg="circle_detector" type="laptop_camera_ros.py" output="screen"/>

    <!-- show image with detected circles -->
    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" respawn="false" output="screen">
        <remap from="image" to="/laptop_camera/image_raw" />
    </node>

</launch>